gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -53,26 +60,36 @@ class BaselineIdea extends AbstractBaselinePlugin {
             }
 
             // Configure Idea module
-            IdeaModel ideaModuleModel = project.extensions.findByType(IdeaModel)
             addJdkVersion(ideaModuleModel)
-            markResourcesDirs(ideaModuleModel);
-            moveProjectReferencesToEnd(ideaModuleModel);
+            markResourcesDirs(ideaModuleModel)
+            moveProjectReferencesToEnd(ideaModuleModel)
         }
 
         // If someone renames a project, leftover {ipr,iml,ipr} files may still exist on disk and
         // confuse users, so we proactively clean them up. Intentionally using an Action<Task> to allow up-to-dateness.
         Action<Task> cleanup = new Action<Task>() {
             void execute(Task t) {
+                if (t.project.rootProject == t.project) {
```

Comment:
Why do we still need this if you're accurately tracking the outputs of the idea task?

Reply:
the ipr and iws tasks only exist on the root project

Reply:
remember that this plugin is applied on all projects